tests/fxl-page-spread_center/OPS/page_001.xhtml

```diff
@@ -0,0 +1,10 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>rendition:page-spread- center</title>
+    <meta name="viewport" content="width=900, height=600" />
+    <link rel="stylesheet" href="fixed.css" />
+  </head>
+  <body id="p1">
+  	<p>Page 1. This test passes if this page is displayed in the center of the screen, not in a synthetic spread.</p>
+  </body>
+</html>
```

Comment:
I have no experience with FXL, so this question may be a throw-away. However, I tried this test on calibre, Apple Books, and Thorium, and none of these implement this properly, at least based on what I see: the colorful pages are either on the left or the right. Is it what we "expect" to see at this point, or is there a problem with the test?